src/Codeception/Step.php

```diff
@@ -94,7 +96,7 @@ public function getArguments()
         return $this->arguments;
     }
 
-    public function getArgumentsAsString($maxLength = 200)
+    public function getArgumentsAsString($maxLength = self::ARGUMENTS_DEFAULT_LENGTH)
```

Comment:
I was going to merge this PR, but now I really don't like the name of this constant.

Reply:
How about `DEFAULT_MAX_LENGTH`?
I've never seen word `ARGUMENTS` used in the name of constant,
if I'm missing some standard or convention, please let me know about it.

Reply:
I renamed it myself.